pages/public_cloud/public_cloud_network_services/technical-resources-02-octavia-monitoring-prometheus/guide.en-gb.md

```diff
@@ -0,0 +1,125 @@
+
+```bash
+$ openstack loadbalancer listener create --name stats-listener --protocol PROMETHEUS --protocol-port 8088 lb1
+
++-----------------------------+--------------------------------------+
+| Field                       | Value                                |
++-----------------------------+--------------------------------------+
+| admin_state_up              | True                                 |
+| connection_limit            | -1                                   |
+| created_at                  | 2023-11-30T01:44:25                  |
+| default_pool_id             | None                                 |
+| default_tls_container_ref   | None                                 |
+| description                 |                                      |
+| id                          | fb57d764-470a-4b6b-8820-627452f55b96 |
+| insert_headers              | None                                 |
+| l7policies                  |                                      |
+| loadbalancers               | b081ed89-f6f8-48cb-a498-5e12705e2cf9 |
+| name                        | stats-listener                       |
+| operating_status            | OFFLINE                              |
+| project_id                  | 4c1caeee063747f8878f007d1a323b2f     |
+| protocol                    | PROMETHEUS                           |
+| protocol_port               | 8088                                 |
+| provisioning_status         | PENDING_CREATE                       |
+| sni_container_refs          | []                                   |
+| timeout_client_data         | 50000                                |
+| timeout_member_connect      | 5000                                 |
+| timeout_member_data         | 50000                                |
+| timeout_tcp_inspect         | 0                                    |
+| updated_at                  | None                                 |
+| client_ca_tls_container_ref | None                                 |
+| client_authentication       | NONE                                 |
+| client_crl_container_ref    | None                                 |
+| allowed_cidrs               | None                                 |
+| tls_ciphers                 | None                                 |
+| tls_versions                | None                                 |
+| alpn_protocols              | None                                 |
+| tags                        |                                      |
++-----------------------------+--------------------------------------+
+```
+
+Once the `PROMETHEUS` listener is `ACTIVE`, you can configure Prometheus to collect metrics from the Load Balancer by updating the `prometheus.yml` file.
+
+```yaml
+[scrape_configs]
+- job_name: 'Octavia LB1'
+  static_configs:
+  - targets: ['192.0.2.10:8088']
+```
+
+For more information on setting up Prometheus, see the [Prometheus project website](https://prometheus.io/).
+
+> [!primary]
```

Comment:
To be removed. This part of the documentation is for the administrators (OVHCloud) , not for the end user